cool-demos/conditional-blocks.html

```diff
@@ -0,0 +1,452 @@
+<!--
+    This demo shows how to use Web Components to create conditional blocks
+    often found in document and marketing automation apps.
+
+    The power with Web Components is that they allow us to create complex
+    HTML components and abstract away that into a single custom element,
+    in this case the <conditional-block>. Take a look at the clean
+    strucured HTML that gets exported out of TinyMCE (Press the last
+    button in the toolbar to view the HTML).
+-->
+
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Conditional blocks</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5-dev/tinymce.min.js" referrerpolicy="origin"></script>
+
+    <script>
+        tinymce.init({
+            selector: "#editor",
+
+            // Tip! To make TinyMCE leaner, only include the plugins you actually need.
+            plugins: "autoresize advcode link lists noneditable",
+
+            // We have put our custom insert conditional block button last.
+            toolbar: "bold italic strikethrough backcolor | bullist numlist link | conditionalblock | code",
+
+            // "Thin" is a premium icon pack you get wth the cloud essential plan.
+            // See https://www.tiny.cloud for more details and purchasing options
+            // You can also create your own icon packs for TinyMCE
+            // https://www.tiny.cloud/docs/advanced/creating-an-icon-pack/
+            icons: 'thin',
```

Comment:
There's a fair bit of inconsistency with quotes here... in some cases it uses single quotes and others it uses double quotes.